android/src/main/java/com/opentokreactnative/OTSessionManager.java

```diff
@@ -218,6 +220,18 @@ public void subscribeToStream(String streamId, String sessionId, ReadableMap pro
         mSubscriber.setStreamListener(this);
         mSubscriber.setSubscribeToAudio(properties.getBoolean("subscribeToAudio"));
         mSubscriber.setSubscribeToVideo(properties.getBoolean("subscribeToVideo"));
+        if (properties.hasKey("preferredFrameRate")) {
+            mSubscriber.setPreferredFrameRate((float) properties.getDouble("preferredFrameRate"));
```

Comment:
Can this casting fail? Do you think we should cast and then try or is this a safe enough approach?

Reply:
The problem is that ReadableMap has only [getDouble](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/bridge/ReadableMap.java#L28) or [getInt](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/bridge/ReadableMap.java#L30).

So, can the cast from `double` to `float` fail?

Reply:
Yes, I think you're good. I was just overthinking it.